src/app/config.rs

```diff
@@ -40,11 +40,21 @@ pub struct Config {
     /// The maximum amount of time to wait for a connection to a remote peer.
     pub outbound_connect_timeout: Duration,
 
-    /// The amount of time to wait between connection attempts.
-    pub inbound_connect_backoff: Duration,
+    /// Settings for the exponential back-off used to determine the time the
+    /// amount of time to wait between connection attempts.
+    pub inbound_connect_exp_backoff_min: Duration,
 
-    /// The amount of time to wait between connection attempts.
-    pub outbound_connect_backoff: Duration,
+    pub inbound_connect_exp_backoff_max: Duration,
+
+    pub inbound_connect_exp_backofff_max_jitter: f64,
```

Comment:
TIOLI: How about we call this `inbound_connect_exp_backoff_jitter`? Since we don't have a min/max here, the "max" may be unnecessary.